src/main/java/com/genpact/insurance/model/Nominee.java

```diff
@@ -4,7 +4,7 @@
 import lombok.*;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
-
+import javax.persistence.*;
```

Comment:
why javax.persistence?

Reply:
using javax.persistence for using @GeneratedValue(strategy = GenerationType.IDENTITY) to generate serial number on it's own.